routes/views/accounts/post/report.js

```diff
@@ -22,7 +22,11 @@ exports = module.exports = async function (req, res) {
   let overallRes = res;
 
   // validate the input
-  req.checkBody('offenders', 'Please indicate the player or players you\'re reporting').notEmpty();
+  let i = 0;
+  while(req.body["offender_"+i]){
```

Comment:
I'm no JS expert, but wouldn't it be less ugly to do something like req.body.filter(startswith("offender_").foreach() ? i-index and while-loop looks so 1980 😃

Reply:
I don't think I can get the i with a foreach. I've fiddled around a bit and For does what i want now :)